.stylelintrc.js

```diff
@@ -6,7 +6,8 @@ module.exports = {
     "LICENSE",
     "*.*",
     "images/*.*",
-    "!*.css"
+    "!*.css",
+    "summarizer/*.txt"
```

Comment:
This change makes me think we need to file a new issue to adjust how our `stylelint` works, such that it only looks at files in a particular directory, maybe `src/frontend/styles/*` or something, and then we don't collide with random files like this.

Reply:
Posted as issue #162.

Reply:
I closed the issue as duplicate. Did not see the issue you posted.

Reply:
As you know, this is being fixed in #151, so why don't you reset your changes to this file, so you don't have merge conflicts with that:

```
git checkout master .stylelintrc.js
```

And you can go back to what it was.  It will fail Travis, but we can ignore.  By the way, you might want to add another review to #151 so we can land it.